src/libsystemd/sd-netlink/netlink-util.c

```diff
@@ -161,6 +161,17 @@ bool rtnl_message_type_is_addr(uint16_t type) {
         }
 }
 
+bool rtnl_message_type_is_addrlabel(uint16_t type) {
+        switch (type) {
+                case RTM_NEWADDRLABEL:
+                case RTM_DELADDRLABEL:
+                case RTM_GETADDRLABEL:
+                        return true;
+                default:
+                        return false;
+        }
+}
```

Comment:
this can be simplified further: please write this as:

```
bool rtnl_message_type_is_addrlabel(uint16_t type) {
        return IN_SET(type, RTM_NEWADDRLABEL, RTM_DELADDRLABEL, RTM_GETADDRLABEL);
}
```

In fact, you could even move this to the header, and make it "static inline" there...